corehq/apps/domain/forms.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, *args, **kwargs):
                     type="submit",
                     css_id="update-proj-settings",
                     css_class='btn-primary',
-                    data_bind="hqbSubmitReady: form_is_ready"
+                    data_bind="hqbSubmitReady: form_is_ready, disable: no_domain_membership"
```

Comment:
I think this is confusing because both `hqbSubmitReady` and `disable` manipulate whether or not the button is disabled. Can you combine these into one binding? You could probably do it with a computed observable that would check the values of both form_is_ready and no_domain_membership.

My bias is towards using just a `disable` binding and getting rid of the `hqbSubmitReady` binding (because we only use `hqbSubmitReady` in one other place so I'd like to kill it).